<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryAttachmentsWidget</class>
 <widget class="QWidget" name="EntryAttachmentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="attachmentsView">
     <property name="accessibleName">
      <string>Attachments</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="actionsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addAttachmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="accessibleName">
         <string>Add new attachment</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeAttachmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="accessibleName">
         <string>Remove selected attachment</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="renameAttachmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="accessibleName">
         <string>Rename selected attachment</string>
        </property>
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openAttachmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="accessibleName">
         <string>Open selected attachment</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAttachmentButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="accessibleName">
         <string>Save selected attachment to disk</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>173</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
